Echo/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ set( PSYNC_HOME
     )
 
```

Comment:
Do the other examples have an install rule in their CMakeLists.txt? I'm used to having the option to run `make install` to put it in the Polysync install's bin directory so we might add that.